test/hotspot/jtreg/vmTestbase/gc/gctests/ReferencesGC/WaitForRefProTest.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Wrong place for this test. vmTestbase is old tests, converted from an old testing infrastructure.
See the readme here:  https://github.com/openjdk/jdk/tree/master/test/hotspot/jtreg/vmTestbase
In particular, at the bottom of that page: "New tests must *not* be added into this directory."

This new test belongs in `test/lib-test/jdk/test/whitebox/`.

Reply:
Also, s/Pro/Proc/ in the name of the test.

Reply:
Sounds good, thanks.